provider/ansible.rb

```diff
@@ -280,11 +280,18 @@ def generate_resource_tests(data)
 
       def compile_datasource(data)
         target_folder = data.output_folder
-        name = "#{module_name(data.object)}_facts"
+        name = module_name(data.object)
         data.generate('templates/ansible/facts.erb',
                       File.join(target_folder,
-                                "lib/ansible/modules/cloud/google/#{name}.py"),
+                                "lib/ansible/modules/cloud/google/#{name}_info.py"),
                       self)
+
+        # Generate symlink for old `facts` modules.
+        deprecated_facts_path = File.join(target_folder,
```

Comment:
@rambleraptor In https://github.com/modular-magician/ansible/pull/298/files#diff-e7fed3ffe467a5f088e35a4117b7cbf1 there are also symlinks created for `_facts` modules which were added in Ansible 2.9 (example: `_gcp_appengine_firewall_rule_facts.py` → `gcp_appengine_firewall_rule_info.py`). These should not be there.

Reply:
(See https://github.com/ansible/ansible/pull/60172#issuecomment-518941454)

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/2131

thanks!